packages/unlock-js/src/web3Service.ts

```diff
@@ -975,20 +975,16 @@ export default class Web3Service extends UnlockService {
   /**
    * Get signer for `Password hook contract`
    */
-  async getPasswordHookSigners(
-    params: {
-      lockAddress: string
-      contractAddress: string
-      network: number
-    },
-    signer: ethers.Wallet | ethers.providers.JsonRpcSigner
-  ) {
+  async getPasswordHookSigners(params: {
```

Comment:
web3Service is for read-only method and does not need a signer.

Reply:
Also we don't actually need to pass a provider as web3Service handles the logic of using the right provider based on the network number.